_posts/2019-04-01-vuzix-blade.md

```diff
@@ -58,21 +64,23 @@ Well actually, it's really more like an Android smartwatch you can strap to your
 
 The device we received was a pre-production build, which was assembled by hand.
 This means we can't really say much about what the final hardware will look like, if there will be any changes or if the build quality, which was very solid, will change.
-During our testing it has been through some light and heavy action, like daily office use, running, biking, etc, and still hasn't shown any faults.
+During our testing it has been through some light and heavy action, like daily commute use, office use, running, biking, etc, and still hasn't shown any faults or cracks.
+
+
+We've have always found the idea of computing devices in the form factor of glasses quite intriguing as some of us have been cursed with nearsightedness and already have to wear prescription glasses.
 
-We've have always found the idea of computing devices in the form factor of glasses quite intriguing as some of us have been cursed with nearsightedness and we have to wear prescription glasses constantly.
 If we have to wear the bloody things every day, might as well put some intelligence into them.
 
 Below you can find some specs about the device, but for us these are quite irrelevant for the moment.
 This device is all about showcasing innovation in two areas: form factor and display technology.
 
 And boy are we impressed.
-The glasses actually feel comfortable enough to wear for longer periods and the display technology is quite amazing!
-It's not HoloLens levels of crazy but still very very good!
+The glasses actually feel comfortable enough to wear for longer periods and the display technology is quite amazing! 
+It's nowhere near the HoloLens, but they serve a completely different purpose.
 
-But let's get down the mandatory spec overview!
+Let's get down the mandatory spec overview!
 The internals inside the glasses are alright, maybe a bit underwhelming.
-But it's always a fine line to balance between power consumption and battery life.
+It's always a fine line to balance between power consumption and battery life.
```

Comment:
Knowing it's always a fine line to balance between power consumption and battery life, the internals inside the glasses are alright, maybe a bit underwhelming.